browser/resources/settings/brave_routes.ts

```diff
@@ -81,6 +81,10 @@ export default function addBraveRoutes(r: Partial<SettingsRoutes>) {
   } else if (!isGuest) {
     console.error('[Brave Settings Overrides] Could not move autofill route to advanced route', r)
   }
+  // Deleted performance menu and system menu includes it instead.
+  if (r.PERFORMANCE) {
+    delete r.PERFORMANCE
```

Comment:
instead of deleting this, can we make it resolve to system? That might solve the failing chromium tests

Reply:
Well, I think it's difficult because we want to merge performance page into the part of system page.
And I think unittest failing is not related with webui changes.

Reply:
Ah, you mean redirecting brave://settings/performance to brave://settings/system ?

Reply:
Confirmed that upstream tests are failed with only features default value change w/o any other changes.

Reply:
ah - there we go! That's the root cause. Thanks for verifying 😄 Will mark this comment as resolved